src/main/resources/db/migration/V1__tables.sql

```diff
@@ -6,52 +6,58 @@ DROP TABLE IF EXISTS versions CASCADE;
 
 CREATE TABLE objects
 (
-    id         BIGSERIAL PRIMARY KEY,
-    hash       CHAR(64) NOT NULL,
-    url        TEXT     NOT NULL,
-    client_id  TEXT     NOT NULL,
-    content    BYTEA    NOT NULL,
-    is_deleted BOOLEAN  NOT NULL DEFAULT FALSE
+    id         BIGINT  GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    hash       TEXT    NOT NULL,
+    url        TEXT    NOT NULL,
+    client_id  TEXT    NOT NULL,
+    content    BYTEA   NOT NULL,
+    is_deleted BOOLEAN NOT NULL DEFAULT FALSE,
+    CONSTRAINT hash_is_unique UNIQUE (hash)
```

Comment:
Do we still have the hash index?

Reply:
Yes, unique constraints are implemented using a unique index.

Reply:
I mean, we still need to have an index for querying by hash. 
Will this automagic index be used for queries?

Reply:
I've used the `CREATE UNIQUE INDEX` syntax again for clarity and to stress that we care mosre about the performance given by an index, instead of the uniqueness (which we ensure by using `INSERT ... ON CONFLICT` anyway).